### PR TITLE
Release images rework, additions, 2.7 issues

### PR DESCRIPTION
**Pull request by swopebe**
@philipwu08 for this issue, we needed to add info about pausing, but we saw there was 1. duplicate files, 2. outdated information, 3. No introduction, 4. Info that has not been revisited in a while. @jnpacker recommended we review these three sections.

### CODE REVIEW

File: clusters/cluster_lifecycle/release_image_connected.adoc [hunk 1]
```diff
@@ -1,58 +1,42 @@
...
 +
 ----
 oc get clusterImageSets
 oc delete clusterImageSet <clusterImageSet_NAME>
 ----
-+
-*Note:* If you have not disabled the installer-managed automatic updates of the `ClusterImageSets` by setting the value of `disableUpdateClusterImageSets` to `true` in the `multiclusterhub` resource, any images that you delete are recreated automatically.
 
-. View the list of currently available release images in the {product-title-short} console when you are creating a cluster.
+View the list of currently available release images in the {product-title-short} console when you are creating a cluster.
```
Comment (swopebe) [inline clusters/cluster_lifecycle/release_image_connected.adoc]: @philipwu08 we keep talking about the rhacm console here:

in the {product-title-short} console

But this content/folder/section is for MCE only

File: clusters/cluster_lifecycle/release_image_intro.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+In {mce-short}, a cluster image set controller that runs on the hub cluster. This controller queries the https://github.com/stolostron/acm-hive-openshift-releases[acm-hive-openshift-releases GitHub repository], at set intervals, for new `clusterImageSets` YAML files. By default, the controller synchronizes with the `fast` channel in the `backplane-2.2` branch.
+
+You can configure your `ClusterImageSets` with the following options. 
+
+- *Option 1:* Specify the image reference for the specific `ClusterImageSet` that you want to use in the console when creating a cluster. Each new entry you specify persists and is available for all future cluster provisions. See the following example entry:
```
Comment (swopebe) [inline clusters/cluster_lifecycle/release_image_intro.adoc:12]: We repeat this content.

File: clusters/cluster_lifecycle/release_images.adoc [hunk 3]
```diff
@@ -28,34 +29,31 @@ spec:
...
+[#config-cluster-image-set]
+== Configure _ClusterImageSets_
 
-**Option 2:** Manually create and apply a `ClusterImageSets` YAML file from the `acm-hive-openshift-releases` GitHub repository.
+You can configure your `ClusterImageSets` with the following options. 
 
-**Option 3:** Follow the `README.md` in the `acm-hive-openshift-releases` GitHub repository to enable automatic updates of `ClusterImageSets` from a forked GitHub repository.
-
-The `subscription` directory contains files that specify where the list of release images is pulled from.
+- *Option 1:* Specify the image reference for the specific `ClusterImageSet` that you want to use in the console when creating a cluster. Each new entry you specify persists and is available for all future cluster provisions. See the following example entry:
```
Comment (swopebe) [inline clusters/cluster_lifecycle/release_images.adoc:48]: repeated content.

**Comment by swopebe** [comment]
I am still waiting for reviews: @berenss @jnpacker 

@philipwu08 provided updates in his PR...I edited quite a bit but we have RHACM references and some repeated content @philipwu08 (I made a couple comments)

File: clusters/cluster_lifecycle/release_image_connected.adoc (hunk 1 above)
Comment (philipwu08) [reply clusters/cluster_lifecycle/release_image_connected.adoc]: Agree, we should remove this and any other reference to the rhacm concole.

**Comment by philipwu08** [comment]
@swopebe The edits look good. Thanks for removing the duplicate. We can remove any ref to the rhacm console. Thanks.

**Pull request merged**